resources/src/main/java/org/robolectric/res/android/ResourceString.java

```diff
@@ -31,14 +31,14 @@
 public final class ResourceString {
 
   /** Type of {@link ResourceString} to encode / decode. */
-  public enum Type {
+  public enum UnicodeType {
```

Comment:
Do we must rename `Type` to `UnicodeType` to fix this warning?

Reply:
We can avoid it also, but I think UnicodeType makes it more clear. It tells more about the enum before even looking into it.

Reply:
it gives more context to the client of this enum, I beleive

Reply:
You can send a new PR for it, if you think it's better. This PR can focus on fixing "Bad import warning".

Reply:
ah okay,I see. make sense. Sure for now going with the `ResourceString.Type` change this variable in the new PR

Reply:
After reverting related changes, please squash all commits to one.

Reply:
Sure :)